Source/WEB_ROOT/admin/asset/js/addHistory.js

```diff
@@ -0,0 +1,22 @@
+function addRecordToHistoryDB(assetid,note)
+    {
+        //console.log("Add new - " + assetid);
+        $psq('u_assethistory').insert(
+            {
+                Asset_ID: assetid,
+                History_DateTime: curDate(),
+                Notes: note
+            });
+
+    }
+
+function curDate()

```

Comment:
I wouldn't go nuts yet. But... if you find yourself using the curDate() function again. It may be worth throwing it in a little helper javascript file. Allowing you to reference it from other scripts without having to rewrite it.